docs/deployment.md

```diff
@@ -133,6 +133,8 @@ uvicorn.run(app, host="127.0.0.1", port=5000, log_level="info")
 However, this style only works if you are not using multiprocessing (`workers=NUM`)
 or reloading (`reload=True`), so we recommend using the import string style.
 
+Also note if you are using reloading(`reload=True`), you should put `uvicorn.run` into `if __name__ == '__main__'` clause in the main module to protect your "entry point".
```

Comment:
I haven't seen `debug` option in docs, maybe it's an outdated option? I just noticed `workers` option also need to attend at this position. It will produce same error message when it's not wrapped by a if-name-main clause.

Reply:
> I haven't seen `debug` option in docs, maybe it's an outdated option? I just noticed `workers` option also need to attend at this position. It will produce same error message when it's not wrapped by a if-name-main clause.

whaaat ! https://github.com/encode/uvicorn/blob/master/uvicorn/main.py#L69-L71

mmm wondering why we dont see it in usage docs....

Reply:
ok debug was added in 546a60496d886f78d9f4dc60bd0f4aa15484f706 and was always a hidden command, let's keep it that way